src/ui/screenshot_menu.css

```diff
@@ -0,0 +1,576 @@
+  font-size: 0.938rem;
+  font-weight: 590;
+  margin: 0;
+}
+.neuroglancer-screenshot-scale-menu {
+  padding: 0 1rem;
+  display: flex;
+  flex-wrap: wrap;
+}
+.neuroglancer-screenshot-scale-menu label {
+  color: var(--gray700);
+  font-size: 0.75rem;
+  margin-right: 2.125rem;
+}
+.neuroglancer-screenshot-scale-menu .neuroglancer-screenshot-scale-radio {
+  margin: 0 0.188rem 0 0;
+}
+
+.neuroglancer-screenshot-keep-slice-fov-checkbox {
+  margin: 0;
+}
+
+.neuroglancer-screenshot-size-text,
+.neuroglancer-screenshot-keep-slice-label,
+.neuroglancer-screenshot-scale-menu .neuroglancer-screenshot-scale-factor {
+  font-size: 0.813rem;
+  color: var(--gray200);
+  font-weight: 590;
+}
+.neuroglancer-screenshot-scale-factor {
+  width: 100%;
+  margin-bottom: 0.5rem;
+  display: flex;
+  align-items: center;
+  gap: 0.15rem;
+}
+.neuroglancer-screenshot-keep-slice-label {
+  display: flex;
+  flex-direction: row-reverse;
+  margin: 1rem 0;
+  width: 100%;
+  justify-content: flex-end;
+  align-items: center;
+  gap: 0.5rem;
+}
+.neuroglancer-screenshot-filename-and-buttons
+  .neuroglancer-screenshot-title-subheading {
+  font-size: 0.938rem;
+  font-weight: 590;
+  margin: 0;
+  display: flex;
+  align-items: center;
+  color: var(--gray600);
+  justify-content: space-between;
+}
+
+.neuroglancer-screenshot-scale-radio {
+  display: inline-block;
+  cursor: pointer;
+}
+
+.neuroglancer-screenshot-filename-and-buttons {
+  padding: 1rem 1rem 0.75rem 1rem;
+}
+.neuroglancer-screenshot-filename-and-buttons label {
+  color: var(--gray800);
+  font-size: 0.813rem;
+  font-style: normal;
+  font-weight: 590;
+  display: block;
+  margin: 0.75rem 0 0.375rem;
+}
+.neuroglancer-screenshot-name-input {
+  width: 100%;
+  margin-right: 10px;
+  border-radius: 8px;
+  border: 1px solid var(--gray300);
+  background: white;
+  font-size: 0.813rem;
+  font-style: normal;
+  padding: 8px 12px;
+  align-items: center;
+  gap: 8px;
+  font-weight: 400;
+  color: var(--gray700);
+  box-shadow: 0px 1px 2px 0px rgba(16, 24, 40, 0.05);
+  line-height: 20px;
+}
+.neuroglancer-screenshot-name-input:disabled {
+  background: var(--gray500);
+  color: var(--gray400);
+}
+.neuroglancer-screenshot-name-input::placeholder {
+  color: var(--gray700);
+}
+
+.neuroglancer-screenshot-button {
+  cursor: pointer;
+  margin: 2px 0px;
+  padding: 0;
+}
+
+.neuroglancer-screenshot-close-button {
+  margin-left: auto;
+  background-color: transparent;
+  border: 0;
+}
+
+.neuroglancer-screenshot-statistics-title {
+  padding: 1rem;
+}
+
+.neuroglancer-screenshot-statistics-table {
+  width: 100%;
+  border-collapse: collapse;
+}
+
+.neuroglancer-screenshot-statistics-table th,
+.neuroglancer-screenshot-statistics-table td {
+  text-align: left;
+  width: 33.33%;
+}
+.neuroglancer-screenshot-statistics-table td {
+  color: var(--gray200);
+  font-size: 0.813rem;
+  font-weight: 590;
+  padding: 0.5rem 0;
+}
+.neuroglancer-screenshot-statistics-table th[colspan="2"] {
```

Comment:
Some of these selectors are getting rather complicated --- a class would probably be clearer.

Reply:
Hopefully this is clearer after the refactor, happy to refine further if helpful